docs/frameworks/nitrojs.md

```diff
@@ -0,0 +1,209 @@
+---
+description: Learn how to deploy a Nitro.js application with Genezio.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+# Nitro.js
+<head>
+    <title>Nitro.js | Genezio Documentation</title>
+</head>
+
+Nitro.js is a modern framework for building server-side applications, designed to optimize performance and streamline development.
+It offers a comprehensive set of features for creating web servers and APIs, supporting multiple deployment targets.
+
+## Prerequisites
+
+### 1. Install genezio
+
+Use your preferred package manager to install genezio:
+
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="step1-install-npm">
+      ```
+      npm install genezio -g
+      ```
+      </div>
+  </TabItem>
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="step1-install-pnpm">
+      ```
+      pnpm add -g genezio
+      ```
+      </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="step1-install-yarn">
+      ```
+      yarn add global genezio
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+
+### 2. Ensure you have an Nitro.js App
+
+If you don't have an Express.js app, you can create one using the following steps:
```

Comment:
"Express.js"? I think you meant nitro.js.

Reply:
Ye stupid mistake, thx for observation